libc-top-half/musl/src/misc/dl.c

```diff
@@ -0,0 +1,31 @@
+#include <stdint.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <dlfcn.h>
+
+#define EXPORT __attribute__((visibility("default"))) __attribute__((used))
```

Comment:
Why is this needed here but not for other libc symbols?  Maybe its not needed?

Reply:
That's left over from earlier experiments.  I've removed it and also removed the unneeded `#include`s.